packages/babel-helpers/src/helpers.js

```diff
@@ -537,51 +541,51 @@ helpers.slicedToArrayLoose = template(`
     } else {
       throw new TypeError("Invalid attempt to destructure non-iterable instance");
     }
-  });
+  }
 `);
 
-helpers.taggedTemplateLiteral = template(`
-  (function (strings, raw) {
+helpers.taggedTemplateLiteral = defineHelper(`
+  export default function _taggedTemplateLiteral(strings, raw) {
     return Object.freeze(Object.defineProperties(strings, {
         raw: { value: Object.freeze(raw) }
     }));
-  });
+  }
 `);
 
-helpers.taggedTemplateLiteralLoose = template(`
-  (function (strings, raw) {
+helpers.taggedTemplateLiteralLoose = defineHelper(`
+  export default function _taggedTemplateLiteralLoose(strings, raw) {
     strings.raw = raw;
     return strings;
-  });
+  }
 `);
 
-helpers.temporalRef = template(`
-  (function (val, name, undef) {
+helpers.temporalRef = defineHelper(`
+  export default function _temporalRef(val, name, undef) {
     if (val === undef) {
       throw new ReferenceError(name + " is not defined - temporal dead zone");
     } else {
       return val;
     }
-  })
+  }
 `);
 
-helpers.temporalUndefined = template(`
-  ({})
+helpers.temporalUndefined = defineHelper(`
+  export default {};
 `);
 
-helpers.toArray = template(`
-  (function (arr) {
+helpers.toArray = defineHelper(`
+  export default function _toArray(arr) {
     return Array.isArray(arr) ? arr : Array.from(arr);
-  });
+  }
 `);
 
-helpers.toConsumableArray = template(`
-  (function (arr) {
+helpers.toConsumableArray = defineHelper(`
+  export default function _toConsumableArray(arr) {
     if (Array.isArray(arr)) {
       for (var i = 0, arr2 = Array(arr.length); i < arr.length; i++) arr2[i] = arr[i];
```

Comment:
Not sure I've heard that before. Not really looking to change the helpers in this PR.